drift/views/v1.py

```diff
@@ -334,3 +334,15 @@ def ensure_account_number():
     message = "Validating account number"
     current_app.logger.audit(message, request=request)
     return view_helpers.ensure_account_number(request, current_app.logger)
+
+
+@section.after_app_request
+def ensure_hsts_response(response):
```

Comment:
Suggestion: This might be made global for the whole app (not only v1).

Reply:
Agree! Made the changes to the code!